common/helpers/helpers.py

```diff
@@ -24,3 +24,56 @@ def str_to_bool(s: Optional[str]) -> bool:
     Returns true if and only if s is defined and some variant capitalization of 'yes' or 'true'.
     """
     return s is not None and s.strip().upper() in ['YES', 'TRUE']
+
+
+  
+def dms2deg(dms):
+
+    if dms is None:
+        return None
+
+    d = float(dms[0])
+    m = float(dms[1])
+    s = float(dms[2])
+    sign = dms[3]
+    dd = d + m / 60. + s / 3600.
+
+    if sign == '-':
+        dd *= -1.
+
+    return dd
+
+
+def dms2rad(dms):
```

Comment:
Same here re: typing.

We might not even need these since we're going to be working to be working with decimal degrees, so it's possible we can eliminate them instead of converting into radians and then converting back, which is what we will have to do now, but maybe the equations for interpolation require radians explicitly, in which case, best not try to mess with them.